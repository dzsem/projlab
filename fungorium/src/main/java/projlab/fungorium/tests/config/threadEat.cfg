# Thread Eat
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD MUSHROOMTHREAD 0 0
# 2
ADD MUSHROOMSPORE 0
# 3
ADD MUSHROOMSPORE 0
# 4
ADD MUSHROOMSPORE 0
# 5
ADD MUSHROOMSPORE 0
SET EFFECTGENERATION 2 STUN
SET EFFECTGENERATION 3 STUN
SET EFFECTGENERATION 4 STUN
SET EFFECTGENERATION 5 STUN
# 6
ADD INSECT 1 0