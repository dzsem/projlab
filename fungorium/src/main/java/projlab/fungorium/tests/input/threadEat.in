# Thread Eat
# ------------------------------------------
# Input
# ------------------------------------------
# Insect gets stunned
LOAD config/threadEat.cfg
INSECT 3 EATSPORE
# Thread eats insect
# 4
MUSHROOMTHREAD 1 EATINSECT
# Print states
# ------------------------------------------
SAVE threadEat.out