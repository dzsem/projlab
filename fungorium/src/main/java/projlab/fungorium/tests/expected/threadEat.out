TECTON 0 0 7 1 3
MUSHROOMTHREAD 1 0 0 0 UNCUT SPROUT
MUSHROOMBODY 7 0 5 0 NORMAL 0